### PR TITLE
June 12 post

### PR DESCRIPTION
**Pull request by bcardarella**

### CODE REVIEW

File: source/posts/2015-03-07-embers-june-12-challenge.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+At EmberConf this year Tom and Yehuda promised that Ember 2.0 RC will be
+out on June 12th.
+
+<iframe width="560" height="315"
+src="https://www.youtube.com/embed/o12-90Dm-Qs?t=38m3s" frameborder="0"
+allowfullscreen></iframe>
+
+Its great to have a defined date that Ember 2.0, Ember-CLI 1.0, and
+Ember Data 1.0 will be released. However, there are currently 692
```
Comment (rwjblue) [inline source/posts/2015-03-07-embers-june-12-challenge.md]: Can you rephrase this to make it clear that Ember 2.0 enters beta on June 12?  "will be released" reads to me like the final version will be released on that date...

File: source/posts/2015-03-07-embers-june-12-challenge.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+packages listed on http://emberaddons.com what good is an Ember 2.0 if
+many of the packages out there aren't going to support it (yet)?
+
+At DockYard we're committed to Ember and its future. To that end we are
+also committing to the June 12th date for our Ember addons:
+**ember-validations 2.0**, **ember-easy-form 2.0**, **ember-admin 1.0**,
```
Comment (michaeldupuisjr) [inline source/posts/2015-03-07-embers-june-12-challenge.md]: Link to each?

File: source/posts/2015-03-07-embers-june-12-challenge.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+At EmberConf this year Tom and Yehuda promised that Ember 2.0 RC will be
+out on June 12th.
+
+<iframe width="560" height="315"
+src="https://www.youtube.com/embed/o12-90Dm-Qs?t=38m3s" frameborder="0"
+allowfullscreen></iframe>
+
+Its great to have a defined date that Ember 2.0, Ember-CLI 1.0, and
+Ember Data 1.0 will be released. However, there are currently 692
+packages listed on http://emberaddons.com what good is an Ember 2.0 if
```
Comment (michaeldupuisjr) [inline source/posts/2015-03-07-embers-june-12-challenge.md]: http://emberaddons.com**. What** good is an Ember 2.0 if
